### ISSUE TITLE
Unable to perform spatial calibration 

### ISSUE THREAD

**Issue by aubest** [PlusToolkit/PlusLib#625]
We have finished perform stylus calibration 、 phantom calibration  and temporal calibration  ，But it always shows questions in the picture below when we perform spatial  calibration.
![image](https://user-images.githubusercontent.com/28101192/67929251-d759ef00-fbf7-11e9-82af-fc42f8f734c6.png)
The method we use is VFMvideo +NDI Polaris.

Below is the XML we wrote from the user manual
![image](https://user-images.githubusercontent.com/28101192/67929877-6f0c0d00-fbf9-11e9-915e-412dbad9b5fe.png)
![image](https://user-images.githubusercontent.com/28101192/67929916-85b26400-fbf9-11e9-98f2-faa65171fc32.png)
![image](https://user-images.githubusercontent.com/28101192/67929984-ad093100-fbf9-11e9-9c90-7f7593bb6ed0.png)

Do you have an idea of what we are possibly doing wrong? 
Thank you very much for your time! 😃

**Comment by Sunderlandkyl** [comment]
What version of Plus are you using?
Can you upload your config and log files in .txt format?

**Comment by aubest** [comment]
Thank you for your reply！ 
I've solved the problem. 
Just change  DataSource Type="Tool" Id="Tool" RomFile="NdiToolDefinitions/1S.rom"   to DataSource Type="Tool" Id="Probe" RomFile="NdiToolDefinitions/1S.rom"  
However, I still cannot do spatial calibration because the dot is not recognized.
When I used probe to scan phantom, the dot was always shown as below.
![image](https://user-images.githubusercontent.com/28101192/67995893-d07abd00-fc87-11e9-922a-cf4fa30a5c6d.png)
